coreDriverInterfaceSupported.json

```diff
@@ -6,6 +6,7 @@
     "2.9",
     "2.10",
     "2.11",
-    "2.12"
+    "2.12",
+    "2.13"
```

Comment:
You also need to update `pluginInterfaceSupported.json`. It is checked in the comment for this PR, but the file change here is not there.